### PR TITLE
feature-swagger

### PR DESCRIPTION
**Pull request by pablohincapie**
## Summary
I added connfiguration properties of swagger for documentation the services

## Trello Card
https://trello.com/c/SJnPeiUt/40-swagger

### CODE REVIEW

File: src/main/java/wolox/training/config/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                .apis(RequestHandlerSelectors.any())
+                .paths(PathSelectors.any())
+                .build().apiInfo(apiEndPointsInfo());
+    }
+
+    private ApiInfo apiEndPointsInfo() {
+        return new ApiInfoBuilder().title("Recipes REST API")
+                .description("REST API that retrieves recipes from a list of ingredientes")
+                .contact(
+                        new Contact("Leonel Costa", "https://github.com/naipes0787",
```
Comment (AleFalcon) [inline src/main/java/wolox/training/config/SwaggerConfig.java]: I think that this contact is not you

File: src/main/java/wolox/training/controllers/BookController.java [hunk 2]
```diff
@@ -44,7 +51,7 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
      * Method that find all the information of one entity.
      * @return: a collection type list with the information of book table
      */
-    @GetMapping
+    @GetMapping()
     public Iterable findAll() {
```
Comment (AleFalcon) [inline src/main/java/wolox/training/controllers/BookController.java:56]: you should document all or most of the methods

File: src/main/java/wolox/training/models/Book.java [hunk 3]
```diff
@@ -24,6 +25,7 @@
     @SequenceGenerator(name = "BOOK_SEQ", sequenceName ="BOOK_SEQ")
     private long id;
     @Column
+    @ApiModelProperty(notes ="The book genre: could be horror, comedy,drama,etc.")
     private String genre;
     @NotNull
     @Column(nullable = false)
```
Comment (AleFalcon) [inline src/main/java/wolox/training/models/Book.java:34]: you should document all or most of the atributes

**Review by AleFalcon — CHANGES_REQUESTED** [review]
You will need to document everything that is minimum users

File: src/main/java/wolox/training/config/SwaggerConfig.java (hunk 1 above)
Comment (pablohincapie) [reply src/main/java/wolox/training/config/SwaggerConfig.java]: I changed the information to my own information

File: src/main/java/wolox/training/models/Book.java (hunk 3 above)
Comment (pablohincapie) [reply src/main/java/wolox/training/models/Book.java:34]: I made documentations to all atributes

File: src/main/java/wolox/training/controllers/BookController.java (hunk 2 above)
Comment (pablohincapie) [reply src/main/java/wolox/training/controllers/BookController.java:56]: I made documentantions to all methods

**Review by Gianni-Nicolas — DISMISSED** [review]

**Review by AleFalcon — DISMISSED** [review]
It's okay

**Review by AleFalcon — APPROVED** [review]

**Review by Gianni-Nicolas — APPROVED** [review]

**Pull request merged**